doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -247,6 +247,33 @@ need to force the parsing level of ``y``, as follows.
    Notation "x < y" := (lt x y) (at level 70).
    Notation "x < y < z" := (x < y /\ y < z) (at level 70, y at next level).
 
+To ease factorization, we provide :n:`at existing level`. This
+:n:`@syntax_modifier` can be used when reserving a notation with an
+already reserved prefix.  The :n:`at existing level or @natural`
+variant can be used to provide a default in case no such prefix can be
+found.
```

Comment:
```suggestion
Use :n:`at existing level` when <WHEN TO USE IT> in order to <DO WHATEVER IT DOES>.
:n:`@syntax_modifier` can be used when reserving a notation with an
already reserved prefix.  Use :n:`at existing level or @natural`
variant to provide a default in case no such prefix can be
found.
```

Easier to understand if it tells the reader when and how to use it.
Would you add a direct statement of what this syntax does?  Examples alone may not be clear enough.
Probably reword/drop ":n:`@syntax_modifier` can be used when reserving a notation with an already reserved prefix" to fit with an updated first sentence.

Reply:
Thanks, I tried something along those lines.